### ISSUE TITLE
signed GET requests

### ISSUE THREAD

**Issue by caver456** [ncssar/sartopo_python#60]
Additional info just received: GET requests for secret maps must be signed (this part might always have been the case); team maps are now secret by default; and new throttling / speedbumps have been introduced. So: creating a new issue for signed GET requests, which will also enable the fixing of #50 and #48.

From comments in #50:

[caver456](https://github.com/caver456) commented [on Feb 17, 2023](https://github.com/ncssar/sartopo_python/issues/50#issuecomment-1434736976)
Resolved this 401 mystery with the help of Matt J and Steve K.

basically: send a signed GET request to https://sartopo.com/api/v1/acct//since/0 with json=’’

The discovery that was holding up progress for months was: In the data to be hashed, make sure you include the newline after the expires timestamp string. The hashed data must have a newline after the URL tail, and another newline after the expires timestamp.

That realization should clear the logjam for this issue and for https://github.com/ncssar/sartopo_python/issues/48.

**Comment by caver456** [comment]
testing with all request types looks good (and fails on a secret map, as expected, when removing the signature args); sending to Steve for testing with plans_console etc

**Comment by caver456** [comment]
Steve confirms this is ready to merge.
